single-user-sobigdata/Dockerfile

```diff
@@ -104,8 +105,14 @@ RUN pip install --no-cache-dir \
         import_ipynb \
         opencv-python \
         simpletransformers \
-        shuffle-graph 
+        shuffle-graph \
+        privlib \
+        ash_model \
+        cdlib==0.3.0 \
```

Comment:
i noticed that you have here cdlib versoin 3, but there is also a cdlib[c] version 2.3.0 up in the installation command, i don't know if this could be an isssue